content/en/docs/reference/glossary/master.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Master
+id: master
+date: 2020-04-16
+short_description: >
+  Deprecated term, used as synonym for nodes running the control-plane.
+
+aka:
+tags:
+- fundamental
+---
+ Deprecated term, used as synonym for {{< glossary_tooltip text="nodes" term_id="node" >}} hosting the {{< glossary_tooltip text="control plane" term_id="control-plane" >}}.
+
+<!--more-->
+The term is still being used by some provisioning tools, such as {{< glossary_tooltip text="kubeadm" term_id="kubeadm" >}}, and managed services.
```

Comment:
Maybe mention that the places that continue to use the term “master” use it in areas such as labels where it is less easy to rename?

Reply:
Sure! I was planning to do that, but I thought it would end up being too verbose. I'll try adding little bit more info.